trustme/__init__.py

```diff
@@ -193,28 +193,36 @@ class CA(object):
           your trust store to trust this CA.
 
     """
-    def __init__(self):
+    def __init__(self, parent_cert=None):
+        self.parent_cert = parent_cert
         self._private_key = rsa.generate_private_key(
             public_exponent=65537,
             key_size=_KEY_SIZE,
             backend=default_backend()
         )
 
+        sign_key = self._private_key
+        if self.parent_cert is not None:
+            sign_key = parent_cert._private_key
+
         name = _name(u"Testing CA #" + random_text())
         self._certificate = (
-            _cert_builder_common(name, name, self._private_key.public_key())
```

Comment:
I think here you want `self._private_key`, not `sign_key`, because this fills in the parts of the cert that say "this is the key that I use to sign things"?

Reply:
@njsmith Thanks! That was the issue. I'll make sure to add tests, yes, thanks for the reminder.